### PR TITLE
[14.0][ADD] create stock_lending

### PR DESCRIPTION
**Pull request by i-vyshnevska**

### CODE REVIEW

File: stock_lending/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    has_return_date = fields.Boolean(related="picking_type_id.has_return_date")
```
Comment (rousseldenis) [inline stock_lending/models/stock_picking.py]: Do these fields are typical to 'stock_lending' ???

**Comment by rousseldenis** [comment]
@i-vyshnevska @TDu I think this module name is not correct. At least, the code does not reflect it.

**Comment by i-vyshnevska** [comment]
@rousseldenis  can you suggest a more suitable name?

**Comment by rousseldenis** [comment]
@i-vyshnevska I don't know exactly what is your business case ? Maybe answering that will generate it. :smiley:

**Comment by i-vyshnevska** [comment]
@rousseldenis updated

**Pull request closed without merging**